edifact-generator/inbound-generator.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import os
+from templates import Inbound
+
+parser = argparse.ArgumentParser(prog=__file__, description='Generate inbound EDIFACT interchanges as .dat files')
+parser.add_argument('--count', type=int, help='the number of files to generate - minimum 1 maximum 9999',
+                    default='1', required=False)
+
+args = parser.parse_args()
+
+
+def run():
+    validate_count_argument()
+    create_output_folder()
+    clear_output_folder()
+
+    generate_edifact_files()
+
+    print("Generated " + str(args.count) + " files in ./output/ folder")
+
+
+def generate_edifact_files():
+    for i in range(0, args.count):
+        filename = "./output/" + str(i) + ".dat"
+        file = open(filename, "w")
+        file.write(Inbound(i).create_edifact())
+        file.close()
+
+
+def validate_count_argument():
```

Comment:
what id the arg is not provided? shouldn't there be a message telling used that the arg should be provided?

Reply:
If it's not provided `args.count` defaults to `1`

Reply:
right, I have forgotten about it 👍